pom.xml

```diff
@@ -259,6 +259,12 @@
             <artifactId>json</artifactId>
             <version>20231013</version>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.datatype</groupId>
+            <artifactId>jackson-datatype-json-org</artifactId>
+            <version>2.16.0</version>
```

Comment:
Better to use `${lib.jackson.version}` here so it's in line with the other Jackson dependencies we have. Different versions of Jackson modules have been problematic in the past.